services/skus/datastore.go

```diff
@@ -1026,6 +1036,21 @@ func (pg *Postgres) GetTLV2Creds(ctx context.Context, dbi sqlx.QueryerContext, o
 	return result, nil
 }
 
+// GetCountActiveOrderCreds returns the count of order creds currently active on an order
+func (pg *Postgres) GetCountActiveOrderCreds(ctx context.Context, dbi sqlx.ExtContext, orderID uuid.UUID) (int, error) {
+	query := `
+		select count(1) from time_limited_v2_order_creds
+		where order_id = $1 and now() > valid_from and valid_to > now() group by request_id
```

Comment:
issue: we are using two different times.

Pass time.Now into the function.

Reply:
addressed: https://github.com/brave-intl/bat-go/pull/2281/commits/ff503ecb24c7e7cdd23a13ef491fd7ae647276ed